sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1095,38 +1088,24 @@ export class BlobServiceClient extends StorageClient {
 
 // @public
 export interface BlobServiceProperties {
-    // (undocumented)
     blobAnalyticsLogging?: Logging;
     cors?: CorsRule[];
     defaultServiceVersion?: string;
-    // (undocumented)
     deleteRetentionPolicy?: RetentionPolicy;
-    // (undocumented)
     hourMetrics?: Metrics;
-    // (undocumented)
     minuteMetrics?: Metrics;
-    // (undocumented)
     staticWebsite?: StaticWebsite;
 }
 
 // @public
 export interface BlobServiceStatistics {
-    // (undocumented)
     geoReplication?: GeoReplication;
 }
 
+// Warning: (ae-forgotten-export) The symbol "BlobSetHttpHeadersHeaders" needs to be exported by the entry point index.d.ts
```

Comment:
export the interface to resolve the warnings.

Reply:
Updated.